satpy/tests/reader_tests/test_viirs_sdr.py

```diff
@@ -841,3 +860,44 @@ def test_bounding_box(self):
         lons, lats = r.file_handlers['generic_file'][0].get_bounding_box()
         np.testing.assert_allclose(lons, expected_lons)
         np.testing.assert_allclose(lats, expected_lats)
+
+
+class FakeTruncatedHDF5FileHandlerAggr(FakeHDF5FileHandler2):
```

Comment:
Just a vocabulary clarification here :) for me "Truncated" sounds like the file was cut off where it shouldn't, but 47 scans per granule is a possible value when the data needs to be adjusted to keep the granules in time. So do I get the meaning of that word wrong, or should we change it?

Reply:
:thinking: There is maybe a better word. I think of this as a truncated granule not necessarily the file. The granule is short because it is not the full/typical 48 scans. I could call it "short"?

Reply:
"short"/"shorter" wfm

Reply:
Done